File: iniciante/2455/2455.c
// 2455 - Gangorra - beecrowd

#include <stdio.h>

int main(void)
{
    int P1, C1, P2, C2;
    scanf("%d %d %d %d", &P1, &C1, &P2, &C2);

    int resultado = P1 * C1 - P2 * C2;

    if (resultado == 0) {
        printf("0\n");
    } else {
        printf("%d\n", (resultado < 0) ? 1 : -1);
    }

    return 0;
}

File: iniciante/2455/2455.py
# 2455 - Gangorra - beecrowd

p1, c1, p2, c2 = map(int, input().split())

resultado = p1 * c1 - p2 * c2

if resultado == 0:
    print(0);
else:
    print(1 if resultado < 0 else -1)
